.circleci/config.yml

```diff
@@ -16,3 +16,17 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+
+      - architect/push-to-app-collection:
+          name: openstack-app-collection
+          context: "architect"
+          app_name: "cluster-api"
+          app_collection_repo: "openstack-app-collection"
+          requires:
+            - push-cluster-api-to-control-plane-catalog
```

Comment:
This change is breaking the builds https://app.circleci.com/pipelines/github/giantswarm/cluster-api-app/55/workflows/6cc781f5-3667-47e1-b1c0-420bf3372c95/jobs/55